contracts/script/test/DepositUsdtFromPR1ToL2.s.sol

```diff
@@ -0,0 +1,37 @@
+// SPDX-License-Identifier: UNLICENSED
+
+pragma solidity ^0.8.25;
+
+import { console } from "forge-std/console.sol";
+import { Script } from "forge-std/Script.sol";
+
+import { T1StandardERC20 } from "../../src/libraries/token/T1StandardERC20.sol";
+import { IL1GatewayRouter } from "../../src/L1/gateways/IL1GatewayRouter.sol";
+
+// solhint-disable var-name-mixedcase
+
+contract DepositUsdtFromPR1ToL2 is Script {
```

Comment:
I don't understand the purpose of this. Why would we want to use the canonical bridge from PR1 to 𝚝𝟷?

Reply:
I can remove this.